### DISCUSSION TITLE
How is the `index` frontmatter implemented?

### DISCUSSION THREAD

**Discussion by ZYJLiu** [fuma-nama/fumadocs#684] [Q&A] [answered]
On fumadocs site, some `index.mdx` files have an [index: true](https://github.com/fuma-nama/fumadocs/blob/main/apps/docs/content/docs/headless/components/index.mdx?plain=1#L3) frontmatter, which seems to be added in [source.ts](https://github.com/fuma-nama/fumadocs/blob/main/apps/docs/utils/source.ts#L12).

It seems that the frontmatter generates these cards links for pages in the folder. Where can I find where this `index` frontmatter is parsed to add these cards?
![CleanShot 2024-08-06 at 16 22 35@2x](https://github.com/user-attachments/assets/60b10f7d-c4b9-41cf-9ed3-40bff7b6a1ce)

**Comment by fuma-nama** [comment, marked as answer]
You can see the docs page for the actual implementation
